server/routes/request.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+const router = express.Router();
+const protect = require("../middleware/auth");
+
+const { 
+    createRequest,
+    userRequests,
+    updateAccept
+} = require("../controllers/request");
+
+router.route("/create").post(protect, createRequest);
+router.route("/list").get(protect, userRequests);
+router.route("/accept").put(protect, updateAccept);
```

Comment:
Since this route can also decline a request, `/accept` could be a misleading name for it. Also, `POST` (or `PATCH`, it kind of depends) would be a more appropriate verb for this.

Reply:
done